go/vt/vtgate/planbuilder/testdata/filter_cases.json

```diff
@@ -6587,73 +6587,110 @@
     "query": "select a.textcol1 from user a join user b where a.textcol1 = b.textcol2 group by a.textcol1 having repeat(a.textcol1,sum(a.id)) like \"And%res\"",
     "v3-plan": "unsupported: cross-shard query with aggregates",
     "gen4-plan": {
+      "QueryType": "SELECT",
```

Comment:
I don't understand why this change happened. It seems the json snippets moved around - `Querytype` used to be at the bottom of this JSON, and now it's at the top. 🤷